src/js/hora.js

```diff
@@ -335,196 +383,11 @@ define([
             }
         };
 
-        Hora.navigation = {
```

Comment:
Yea. I'll remove the reference in readme... but how do we deal with the wiki ones?